test/pod_annotations_capability.bats

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+function run_pod() {
+	crictl runp "$TESTDATA"/sandbox_config.json
+}
+
+function delete_pod() {
+	id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+	crictl stopp "$id"
+	crictl rmp "$id"
+}
+
+@test "single cni plugin with pod annotations capability enabled" {
+	enable_capability=true
+	cni_plugin_log_path="$TESTDIR/cni.log"
+
+	start_crio
+	prepare_cni_plugin "$cni_plugin_log_path" $enable_capability
+	run_pod
+	delete_pod
+
+	annotations_cni_plugin_received=$(jq '.runtimeConfig."io.kubernetes.cri.pod-annotations"' "$cni_plugin_log_path")
```

Comment:
```
# jq: error: Could not open file /tmp/tmp.LCZOxyLqP3/cni.log: No such file or directory
```
this line fails in the tests still

Reply:
Yeah, it's strange bc the same test passes locally.